config.py

```diff
@@ -100,6 +100,8 @@ class LocalConfig(BaseConfig):
     # Using a local sqlite database
     SQLALCHEMY_DATABASE_URI = "sqlite:///local_data.db"
 
+    # Using a local postgre database
+    # SQLALCHEMY_DATABASE_URI = "postgresql:///local_data"
```

Comment:
I don't think this is the way to go. I think we should use environment variables for this.